sycl/include/sycl/kernel.hpp

```diff
@@ -137,7 +142,15 @@ class __SYCL_EXPORT kernel : public detail::OwnerLessBase<kernel> {
   ///
   /// \return depends on information being queried.
   template <typename Param>
-  typename detail::is_kernel_info_desc<Param>::return_type get_info() const;
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  typename detail::is_kernel_info_desc<Param>::return_type get_info() const {
+    return detail::convert_from_abi_neutral(get_info_impl<Param>());
+  }
+#else
+  detail::ABINeutralT_t<
```

Comment:
If this is always an alias to `T`, I don't see a point in using it in this path. I would just keep it as-is.

Reply:
Done. Thanks.